boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.dts

```diff
@@ -104,6 +104,24 @@
 	pinctrl-names = "default";
 };
 
+arduino_i2c: &flexcomm4 {
+	compatible = "nxp,lpc-i2c";
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
+	#address-cells = <1>;
+	#size-cells = <0>;
```

Comment:
note: this should be moved to soc dts files

Reply:
If I'm understanding correctly, the #address-cells, #size-cells properties are used for the child node "fxos8700@1e" which is defined below. So they are put here not in soc dts. 
Correct me if I'm wrong. 

Thanks. 
> note: this should be moved to soc dts files